doc/source/rllib/rllib-catalogs.rst

```diff
@@ -4,50 +4,50 @@
 
 .. include:: /_includes/rllib/rlmodules_rollout.rst
 
-.. note:: Interacting with Catalogs mainly covers advanced use cases.
 
-Catalog (Alpha)
+Catalog (Beta)
 ===============
 
 Catalogs are where `RLModules <rllib-rlmodule.html>`__ primarily get their models and action distributions from.
 Each :py:class:`~ray.rllib.core.rl_module.rl_module.RLModule` has its own default
 :py:class:`~ray.rllib.core.models.catalog.Catalog`. For example,
 :py:class:`~ray.rllib.algorithms.ppo.ppo_torch_rl_module.PPOTorchRLModule` has the
 :py:class:`~ray.rllib.algorithms.ppo.ppo_catalog.PPOCatalog`.
-You can override Catalogs’ methods to alter the behavior of existing RLModules.
-This makes Catalogs a means of configuration for RLModules.
```

Comment:
This is misleading.
Users should not randomly land on this page, read the first few sentences and attempt to configure an RL Module through catalogs. This gets clearner further down.